### PR TITLE
Task2

### PR DESCRIPTION
**Pull request by domipepe**

### CODE REVIEW

File: public_html/index.html [hunk 1]
```diff
@@ -18,12 +18,24 @@
       <content>
           <div class="side-panel">
               Boczny panel
+
```
[the remark this answers is not in the document]
Comment (domipepe) [reply public_html/index.html]: 62f54b933e3f01235a85982bb3c1080a7d966357

File: public_html/index.html [hunk 2]
```diff
@@ -18,12 +18,24 @@
...
           </div>
           <div class="main-panel">
               Główny panel
           </div>
           <div style="clear: both;"></div>
       </content>
+      <content>
+          <div class ="cezarek">
```
[the remark this answers is not in the document]
Comment (domipepe) [reply public_html/index.html]: 62f54b933e3f01235a85982bb3c1080a7d966357

File: public_html/index.html [hunk 3]
```diff
@@ -18,12 +18,24 @@
...
+              <img src="ddd.jpg"  width="400" height="330">
```
[the remark this answers is not in the document]
Comment (domipepe) [reply public_html/index.html]: 62f54b933e3f01235a85982bb3c1080a7d966357

**Comment by mjMikolowski** [comment]
#Aprroved

**Pull request merged**